### PR TITLE
`update_users_me` Alias for GraphQL API

### LINKED ISSUE #10500: Add PATCH /users/me to docs and GraphQL
### Preflight Checklist

- [X] I have completed all [Troubleshooting Steps](https://docs.directus.io/getting-started/support/#troubleshooting-steps).
- [X] I'm on [the latest version of Directus](https://github.com/directus/directus/releases).
- [X] There's [no other issue](https://github.com/directus/directus/issues?q=is%3Aissue) that already describes my problem.

### Describe the Bug

We have a /users/me endpoint that can be used to retrieve the current user. However, there's also a (currently undocumented) PATCH /users/me, that allows to easily update the current user. We should make sure this is documented.

While we do that, we should also make sure there's a users_me mutation to align the feature parity between the REST and GraphQL operations.

### To Reproduce

—

### Errors Shown

_No response_

### What version of Directus are you using?

9.2.2

### What version of Node.js are you using?

16

### What database are you using?

Postgres 13

### What browser are you using?

Safari

### What operating system are you using?

macOS Big Sur

### How are you deploying Directus?

Locally

### PR DESCRIPTION
**Pull request by wrynegade**
Resolves #10500

- Created the `mutation` for `update_users_me`
- Added documentation for REST PATCH and GraphQL Mutation on `users_me`

### CODE REVIEW

**Comment by benhaynes** [comment]
Thank you!! Always exciting to see "first-time contributor"! 🧑‍🤝‍🧑🧑‍🤝‍🧑🧑‍🤝‍🧑

File: api/src/services/graphql.ts [hunk 1]
```diff
@@ -1578,6 +1578,7 @@ export class GraphQLService {
 		{
 			CreateCollectionTypes,
 			ReadCollectionTypes,
+			UpdateCollectionTypes,
 			DeleteCollectionTypes,
 		}: {
```
Comment (w0ryn) [inline api/src/services/graphql.ts:1583]: I'm a little concerned about some kind of unknown (to me) cascading effect from this. Any concerns?

Comment (benhaynes) [reply api/src/services/graphql.ts:1583]: That's a question for the actual engineering team 😉

Comment (rijkvanzanten) [reply api/src/services/graphql.ts:1598]: No side effects 👍🏻

**Review by rijkvanzanten — APPROVED** [review]

**Pull request merged**